tests/chainer_tests/functions_tests/math_tests/test_cholesky.py

```diff
@@ -0,0 +1,83 @@
+import numpy
+
+import chainer.functions as F
+from chainer import testing
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+    'shape': [(2, 2)]
```

Comment:
- I'd test larger size.
- If you'd like to be sure with omitting `utils.force_array`, testing `shapes=(1, 1)` looks good to me.
```suggestion
    'shape': [(5, 5), (1, 1)]
```